src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotRight = {};
```

Comment:
What means robotRight? Why not robotLeft? =) reversedRobot is clearer

Reply:
I meant a different meaning, but you're right, your name is better)